<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Fenetre</class>
 <widget class="QDialog" name="Fenetre">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>734</width>
    <height>467</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="AddNoeud">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>30</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>AddNoeud</string>
   </property>
  </widget>
  <widget class="QPushButton" name="AddArc">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>80</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>AddArc</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Solution">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>30</y>
     <width>111</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string>Solution</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Noeuds</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Arcs</string>
   </property>
  </widget>
  <widget class="QLabel" name="GrapheIm">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>701</width>
     <height>291</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>le graphe va s'afficher ici</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="noeudListe">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>30</y>
     <width>281</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="ArcListe">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>80</y>
     <width>281</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
